src/libs/DistanceRequestUtils.ts

```diff
@@ -38,6 +39,43 @@ Onyx.connect({
 const METERS_TO_KM = 0.001; // 1 kilometer is 1000 meters
 const METERS_TO_MILES = 0.000621371; // There are approximately 0.000621371 miles in a meter
 
+/**
+ * Retrieves the mileage rates for given policy.
+ *
+ * @param policy - The policy from which to extract the mileage rates.
+ * @param includeDisableRate - Should include disable rate
+ *
+ * @returns An array of mileage rates or an empty array if not found.
+ */
+function getMileageRates(policy: OnyxEntry<Policy>, includeDisableRate = true): Record<string, MileageRate> {
```

Comment:
Let's please rename it to `includeDisabledRates` since that reads better. I think we should also default it to `false`, because that's more often what we want.

However, it is good to have the param so that we can set it to true when viewing an existing request, like in MoneyRequestView.

Within `calculateAmountForUpdatedWaypoint` I think we want to set it to true as well so that updating the waypoints on an existing request, with a now disabled rate, will continue to use the same rate. That's pretty complicated but could happen 😅